rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -207,15 +207,15 @@ namespace rsx
 		{
 			if (rsx->in_begin_end)
 			{
-				rsx->append_array_element(arg & 0xFFFF);
-				rsx->append_array_element(arg >> 16);
+				rsx->append_array_element(se_storage<u16>::swap(arg & 0xFFFF));
+				rsx->append_array_element(se_storage<u16>::swap(arg >> 16));
```

Comment:
Isn't the result the same?
I mean extracting the relevant 16 bits and swapping them should result in the same value unless I'm misunderstanding the contents of the arg here. From nv40 docs the arg value contains 2 packed 16-bit indices with 0-16 containing the first value and 16-32 the second value.